src/Libraries/Microsoft.Extensions.AI.Evaluation.Console/Microsoft.Extensions.AI.Evaluation.Console.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>A dotnet tool for managing the evaluation data and generating reports.</Description>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>$(NetCoreTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>$(MinimumSupportedTfmForPackaging)</TargetFrameworks>
```

Comment:
It may be a good idea to leave a comment here linking the SDK bug and saying that we can revert back to building all TFMs once it is fixed.